api/Document.json

```diff
@@ -5035,10 +5037,12 @@
               "version_added": true
             },
             "safari": {
-              "version_added": null
+              "version_added": true,
```

Comment:
Can you update this with when `document.webkitFullscreenEnabled` was introduced? I've just done https://github.com/foolip/mdn-bcd-collector/pull/589 to make sure we generate the test for it.